python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -306,20 +308,22 @@ class RandomForestRegressor(Base):
         cdef size_t params_t64
```

Comment:
`rf_forest` and `rf_forest64` should be of type `RandomForestMetaData[float, float]` and `RandomForestMetaData[double double]` respectively